articles/azure/azs-how-configure-mars-backups-powershell.md

```diff
@@ -0,0 +1,91 @@
+---
+title: How to backup file(s) and folder(s) on Azure Stack VMs using Microsoft Azure recovery services (MARS) - PowerShell | UKCloud Ltd
+description: Provides details on how to install and configure the MARS agent to backup file(s) and folder(s) on Azure Stack
+services: azure-stack
+author: Daniel Brennand
+reviewer:
+lastreviewed:
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: How to backup file(s) and folder(s) on Azure Stack VMs using Microsoft Azure recovery services (MARS) - PowerShell
+toc_fullpath: Users/How To/azs-how-configure-mars-backups-powershell.md
+toc_mdlink: azs-how-configure-mars-backups-powershell.md
+---
+
+# Backing up file(s) and folder(s) on Azure Stack VMs using Microsoft Azure recovery services (MARS) - PowerShell
+
+# Overview
+
+This article explains how to setup the MARS agent to backup file(s) and folder(s) from Azure Stack VMs to Microsoft Azure recovery services vaults.
```

Comment:
Spell out the product name in full the first time is it used with the main body of text:
This article explains how to setup the Microsoft Azure Recovery Services (MARS)  agent to backup files and folders from Azure Stack VMs to Microsoft Azure recovery services vaults.